jupyterlab/browser_check.py

```diff
@@ -37,11 +37,12 @@
 test_aliases["app-dir"] = "BrowserApp.app_dir"
 
 
-class LogErrorHandler(logging.Handler):
+class LogErrorHandler(logging.StreamHandler):
     """A handler that exits with 1 on a logged error."""
 
     def __init__(self):
-        super().__init__(level=logging.ERROR)
+        super().__init__()
```

Comment:
To match the previous functionality, should it be:

```suggestion
        super().__init__(stream=sys.stderr)
```

Reply:
I did not set it because it is the [default value](https://docs.python.org/3/library/logging.handlers.html#logging.StreamHandler)

But if you think it is better to explicit it, no problem.